src/doppkit_toolbox/esri/toolboxes/doppkit_wrapper_toolbox.pyt

```diff
@@ -135,7 +180,12 @@ class FetchExport:
         app.pk = aoi_pk
         app.disable_ssl_verification = False
 
-        sync(app, aoi_pk)
+        try:
+            sync(app, aoi_pk)
+        except:
```

Comment:
Don't catch bare exceptions, what's the specific exception you're trying to catch?